src/main/resources/avro/metadata.avdl

```diff
@@ -114,4 +135,42 @@ record Dataset {
 
 }
 
+/**
+An analysis contains an interpretation of one or several experiments.
+(e.g. SNVs, copy number variations, methylation status) together with
+information about the methodology used.
+*/
+record Analysis {
+  /** The analysis UUID. This is globally unique. */
+  string id;
+
+  /** The name of the analysis. */
+  union { null, string } name = null;
+
+  /** A description of the analysis. */
+  union { null, string } description = null;
+
+  /**
+  The time at which this record was created. 
+  Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
+  */
+  union { null, string } recordCreateTime = null;
+
+  /**
+  The time at which this record was last updated.
+  Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
+  */
+  string recordUpdateTime;
+
+  /** The type of analysis. */
+  union { null, string } type = null;
+
+  /** The software run to generate this analysis. */
+  array<string> software = [];
```

Comment:
Yes, please move program to metadata.

Sarah Hunt <notifications@github.com> writes:
> In src/main/resources/avro/metadata.avdl:
> 
> > +  The time at which this record was created.
> > +  Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
> > +  */
> > +  union { null, string } recordCreateTime = null;
> > +
> > +  /**
> > +  The time at which this record was last updated.
> > +  Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
> > +  */
> > +  string recordUpdateTime;
> > +
> > +  /** The type of analysis. */
> > +  union { null, string } type = null;
> > +
> > +  /** The software run to generate this analysis. */
> > +  array<string> software = [];
> 
> We are just re-using the metadata defined record here. It probably makes sense
> to move Program out of reads.avdl and merge the two.
> 
> —
> Reply to this email directly or view it on GitHub.*
>